pkg/controllers/member/serviceexport/controller.go

```diff
@@ -42,6 +48,8 @@ const (
 
 	// ControllerName is the name of the Reconciler.
 	ControllerName = "serviceexport-controller"
+
+	publicLoadBalancerName = "kubernetes"
```

Comment:
Do we only support AKS cluster for now or any clusters held in Azure? Another thing is looks like multi-SLB work is undergoing, so we will need to dynamically decide LB name from service. For now, it's ok.

Multi-SLB info will be included in azure cloud config too: https://github.com/kubernetes-sigs/cloud-provider-azure/blob/master/pkg/provider/azure.go#L295